### DISCUSSION TITLE
How to run the example projects?

### DISCUSSION THREAD

**Discussion by hollanderbart** [software-mansion/react-native-screens#2086] [General]
I'm having trouble with starting the example projects. It cannot find the local modules from the root project. 

```
error: Error: Unable to resolve module @react-navigation/native-stack from /Users/dir/react-native-screens/TestsExample/src/Test42.tsx: @react-navigation/native-stack could not be found within the project or in these directories:
  node_modules
  ../node_modules
  ../..
   6 |   NativeStackNavigationProp,
   7 | } from 'react-native-screens/native-stack';
>  8 | import { createNativeStackNavigator } from '@react-navigation/native-stack';
     |                                             ^
   9 | import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
  10 | // import {createStackNavigator} from '@react-navigation/stack';
  11 |
```

I've read the README and ran `yarn install` on both the root directory and example directory.

Thanks!

**Comment by tboba** [comment]
Hi @hollanderbart! Yeah, the error comes because of the latest change with adding react-navigation as a submodule in screens repo. To initialize it, you must execute `git submodule update --init --recursive && (cd react-navigation && yarn)` command (we'll add this command in the future to package.json. It was previously there, but since it was `postinstall` command, it was crashing all projects that had screens bundled 🙈).

Cheers!
